WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -290,15 +281,17 @@ public static void showSignInForResult(Activity activity) {
         }
     }
 
-    public static void viewStatsSinglePostDetails(Context context, SiteModel site, Post post, boolean isPage) {
+    public static void viewStatsSinglePostDetails(Context context, SiteModel site, PostModel post, boolean isPage) {
         if (post == null) return;
 
-        PostModel postModel = new PostModel(site.getSiteId(), post.getRemotePostId(), post.getTitle(), post
-                .getLink(), isPage ? StatsConstants.ITEM_TYPE_PAGE : StatsConstants.ITEM_TYPE_POST);
+        org.wordpress.android.ui.stats.models.PostModel postModel =
```

Comment:
Maybe we should rename `org.wordpress.android.ui.stats.models.PostModel` to StatsPostModel to avoid confusion?